client/app.js

```diff
@@ -92,4 +92,13 @@ porybox.service('io', function () {
   return io;
 });
 
+porybox.service('errorHandler', ['$mdToast', function ($mdToast) {
+  return reason => {
+    console.error(reason);
+    $mdToast.show(
+      $mdToast.simple().textContent('An unexpected error occured.').position('bottom right')
```

Comment:
I'd say it's fine. This is for errors that are outside the user's control anyway, maybe it makes sense for them to be differentiable.